test/framework/integration_framework/fake_peer/ordering_service_network_notifier.cpp

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "framework/integration_framework/fake_peer/ordering_service_network_notifier.hpp"
+
+#include "interfaces/iroha_internal/transaction_batch.hpp"
+
+namespace integration_framework {
+
+  void OsNetworkNotifier::onBatch(std::unique_ptr<TransactionBatch> batch) {
+    auto batch_ptr = std::shared_ptr<TransactionBatch>(batch.release());
```

Comment:
I would rather avoid explicit `release()` call.
```suggestion
    std::shared_ptr<TransactionBatch> batch_ptr = std::move(batch);
```